src/NewsletterEpic/index.tsx

```diff
@@ -97,7 +97,7 @@ const styles = {
     `,
 };
 
-export type NewsletterSubscribeCallback = (id: string) => Promise<void>;
+export type NewsletterSubscribeCallback = (id: string) => Promise<unknown>;
```

Comment:
I'm interested in the change from `Promise<void>` to `Promise<unknown>` here. We own this contract so I don't think `unknown` is quite right, because it _is_ known! (And the interface of the [callback supplied by DCR](https://github.com/guardian/dotcom-rendering/blob/a1d024bc118403e6be0b21ca53201bc85e39eb49/dotcom-rendering/src/web/components/SlotBodyEnd/BrazeEpic.tsx#L130) is in fact `(id: string) => Promise<void>`.

Reply:
Typescript hates me! If you can solve the issue I'd love to see how it's done.